rxpm/src/main/kotlin/me/dmdev/rxpm/validation/FormValidator.kt

```diff
@@ -0,0 +1,95 @@
+package me.dmdev.rxpm.validation
+
+import me.dmdev.rxpm.PresentationModel
+import me.dmdev.rxpm.widget.CheckControl
+import me.dmdev.rxpm.widget.InputControl
+
+
+/**
+ * Use this class to validate the form.
+ * For this, [create][formValidator] FormValidator using DSL, to check the [input fields][input] and [checkbox][check].
```

Comment:
To check the [input fields][input] and [checkbox][check] [create][formValidator] FormValidator using DSL.